README.md

```diff
@@ -1,33 +1,68 @@
-# [ownCloud](https://owncloud.org) iOS app
+# [ownCloud](https://owncloud.org) iOS App
 
-Available in the Apple App Store:
+## 📱 The new iOS app for your ownCloud!
+
+####  🌎 https://ownCloud.com
+
+| Account List                                                 | File List                                                    | File Actions                                                 | Preview Files                                                | Quick Access                                                 | Settings                                                     |
+| ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ | ------------------------------------------------------------ |
+| <img src="fastlane/screenshots/en-US/iPhone Xs Max-11_ios_accounts_list_demo.png" alt="Simulator Screen Shot - iPhone 11 Pro"> | <img src="fastlane/screenshots/en-US/iPhone Xs Max-20_ios_files_list_demo.png" alt="Simulator Screen Shot - iPhone 11 Pro"> | <img src="fastlane/screenshots/en-US/iPhone Xs Max-21_ios_files_actions_demo.png" alt="Simulator Screen Shot - iPhone 11 Pro"> | <img src="fastlane/screenshots/en-US/iPhone Xs Max-22_ios_files_preview_pdf_demo.png" alt="Simulator Screen Shot - iPhone 11 Pro"> | <img src="fastlane/screenshots/en-US/iPhone Xs Max-40_ios_quick_access_demo.png" alt="Simulator Screen Shot - iPhone 11 Pro"> | <img src="fastlane/screenshots/en-US/iPhone Xs Max-60_ios_settings_demo.png" alt="Simulator Screen Shot - iPhone 11 Pro"> |
+
+## 😍 Features
+
+* 🦋 Using the current iOS frameworks
+* ⭐️ Exclusively built for iOS
+* 🛠 Written in Swift
+* 📂 Seamless integration with iOS files for improved collaboration
+* 🕹 Multi-select with drag and drop features for efficient file management
+* 👑 Using iPadOS features for pro users
+* ✅ Certificate management and password manager integration for boosting security
+* 🚢 Free on the AppStore
+* 🇪🇺 Localised in many languages
+* 📱iOS 11+
+* 🧩 License: [GPLv3](https://github.com/owncloud/ios-app/LICENSE)
+
+## 📲 Download
+
+Our iOS app is free for downloading available on the AppStore
 
 [![](https://owncloud.org/wp-content/themes/owncloudorgnew/assets/img/clients/buttons/appstore.png)](https://apps.apple.com/app/id1359583808)
 
+## 🛠 Build our App
 
-## Join development!
+To build our app, please read our [SETUP.md](https://github.com/owncloud/ios-app/blob/master/SETUP.md)
 
-**Build status:**
+```
+$ read [SETUP.md]()
+```
 
```

Comment:
That script to read he SETUP.md is not correctly interpreted

![Screenshot 2020-01-21 at 12 35 33](https://user-images.githubusercontent.com/14894746/72801775-8da70080-3c4a-11ea-92ec-c22a68f16429.png)

Reply:
I removed the brackets, but it should look like a command line shell.

Reply:
Then it is OK